Blocks/MovingBlock36.md

```diff
@@ -1,38 +1,39 @@
 ---
-title: Moving piston block 36
+title: Moving block 36
 description: Mechanics around the moving piston block
 ---
 
-# moving piston block 36
+# Moving piston block 36
```

Comment:
does not have the same change as the title. is that an oversight?

Reply:
Fixed